gcetcbendorsement/cmd/root.go

```diff
@@ -96,15 +97,38 @@ func (b *bearerGetter) Get(url string) ([]byte, error) {
 	return io.ReadAll(resp.Body)
 }
 
+// The go-tdx-guest IsSupported signature is different from go-sev-guest. Add a compatibility layer.
+type tdxQuoteProvider struct{ p tdclient.QuoteProvider }
+
+func (qp *tdxQuoteProvider) IsSupported() bool { return qp.p.IsSupported() == nil }
+func (qp *tdxQuoteProvider) GetRawQuote(reportData [64]byte) ([]uint8, error) {
+	return qp.p.GetRawQuote(reportData)
+}
+func createTdxQuoteProvider() (extract.QuoteProvider, error) {
+	tdp, _ := tdclient.GetQuoteProvider() // Never errors.
+	return &tdxQuoteProvider{p: tdp}, nil
+}
+
+func getProvider() (p extract.QuoteProvider) {
+	// Eta-expand the go-sev-guest creator since Golang doesn't do function type conversion.
+	createSev := func() (extract.QuoteProvider, error) { return client.GetQuoteProvider() }
+	creators := []func() (extract.QuoteProvider, error){createSev, createTdxQuoteProvider}
```

Comment:
nit: Consider removing the error return since it's ignored and you have to define stubs anyway